scripts/cluster/images.py

```diff
@@ -25,5 +30,32 @@ def import_images(image: str):
     do_image_import(image_data)
 
 
+def get_all_ctr_images():
+    """
+    Return list of all OCI images from containerd.
+    """
+    images = subprocess.check_output([CTR, "image", "ls", "--quiet"]).decode().split("\n")
```

Comment:
Looking at this output:
```
$ microk8s images export > images.tar
Checking docker.io/calico/cni:v3.21.4
Checking docker.io/calico/cni@sha256:36acb85e6080a3a894ea5f92f20dabd1179061f5ecb079a49913bbdfb714ad09
Checking docker.io/calico/kube-controllers:v3.21.4
Checking docker.io/calico/kube-controllers@sha256:f71a293e43f66c8f3d2435a3d56ed53cbb1bcf539dcfe7fe2cc64d4258f4f792
Checking docker.io/calico/node:v3.21.4
Checking docker.io/calico/node@sha256:acb402642ba8a9a28eda86d9cd3eef97976dd339c11e33863faeae4e00682e12
Checking docker.io/calico/pod2daemon-flexvol:v3.21.4
Checking docker.io/calico/pod2daemon-flexvol@sha256:baeaa86e59194d2707be3b708b5ef157fe09738605bdabffc5a4a7cc7560b7d7
Checking k8s.gcr.io/pause:3.1
Checking k8s.gcr.io/pause@sha256:f78411e19d84a252e53bff71a4407a5686c46983a2c2eeed83929b888179acea
```

Does this export each image twice?

Reply:
it will export all image tags, the resulting tarball will not contain duplicate layers